From 828ce32951cf5d09ceedb4db7d6fbd70dfe383a8 Mon Sep 17 00:00:00 2001
From: Cosimo Cecchi <cosimoc@gnome.org>
Date: Thu, 11 May 2017 16:07:21 -0700
Subject: [PATCH] tilescreen: synchronize definition of MAX_UPDATES

t4kcommon also defines the same, and mixing them leads to a segfault.
---
 src/titlescreen.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/titlescreen.h b/src/titlescreen.h
index 3445907..880b2dc 100644
--- a/src/titlescreen.h
+++ b/src/titlescreen.h
@@ -62,7 +62,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define MAX_WORD_SIZE                   8
 
 //MAX_UPDATES needed for TransWipe() and friends:
-#define MAX_UPDATES                     180
+#define MAX_UPDATES                     512
 
 #define WAIT_MS                         2500
 #define FRAMES_PER_SEC                  50
-- 
2.12.2

